src/Microsoft.VisualStudio.Threading.Analyzers/VSTHRD112AvoidNullReturnInNonAsyncTaskMethodAnalyzer.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Microsoft.VisualStudio.Threading.Analyzers
+{
+    using System.Collections.Immutable;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Microsoft.CodeAnalysis.Operations;
+
+    /// <summary>
+    /// Finds await expressions on <see cref="Task"/> that do not use <see cref="Task.ConfigureAwait(bool)"/>.
+    /// Also works on <see cref="ValueTask"/>.
+    /// </summary>
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
```

Comment:
Can you add VB as a supported language here?

Reply:
Do you want me to introduce vbnet tests?

Reply:
Just one or two sanity tests in VB would be great, yes.